src/Symfony/Component/Console/Completion/CompletionSuggestions.php

```diff
@@ -18,33 +18,37 @@
  *
  * @author Wouter de Jong <wouter@wouterj.nl>
  */
-class CompletionSuggestions
+final class CompletionSuggestions
```

Comment:
We can remove final now, but do we want this?

Reply:
when a class is part of a public API, it's better to not make it final IMHO, to provide extensibiliity

Reply:
As this is a value object that has no reason to be mocked (just use an actual instance), I suggest keeping it fina